docs/edgekv_cli.md

```diff
@@ -392,6 +393,25 @@ Usage:
 | - | - | - |
 | tokenName | required | token name |
 
+### List Groups in Namespace
+
+List the groups for a given namespace in an Akamai environment.
```

Comment:
Change to:
List the data groups for a given namespace in an Akamai environment.

Reply:
done